muesli/web/templates/tutorial/view.pt

```diff
@@ -6,91 +6,119 @@
+  ${names['tutor']}:
+  <a tal:attributes="href 'mailto:'+tutorial.tutor.email">
+    ${tutorial.tutor.name}
+  </a>
 </p>
 
-<h3>Administrativa</h3>
-
-<ul>
-  <li><a tal:attributes="href request.route_path('tutorial_results', lecture_id=tutorial.lecture_id, tutorial_ids=tutorial_ids)">Punkteübersicht</a></li>
-  <li><a tal:attributes="href request.route_path('tutorial_email', tutorial_ids=tutorial_ids)">E-Mail an Teilnehmer schreiben</a></li>
-  <li><a tal:attributes="href request.route_path('tutorial_email_preference', tutorial_ids=tutorial_ids)">Status-Emails bestellen/ abbestellen</a></li>
-  <li><a tal:attributes='href request.route_path("lecture_email_tutors", lecture_id=tutorial.lecture.id)'>E-Mail an alle Übungsleiter und Assistenten schreiben</a></li>
-  <li tal:condition="request.permissionInfo.has_permission('edit') and len(tutorials) &lt; 2">
-    <a tal:attributes="href request.route_path('tutorial_edit', tutorial_id=tutorial.id)">${names['tutorial']} bearbeiten</a></li>
-  <li tal:condition="request.userInfo.is_tutor_of_tutorials(tutorials)">
-     <a tal:attributes="href request.route_path('tutorial_resign_as_tutor', tutorial_ids=tutorial_ids)">Als ${names['tutor']} zurücktreten</a></li>
-</ul>
-
-<h3>Testate</h3>
-
-<div tal:repeat="category categories" tal:omit-tag="">
-  <h4 tal:condition="exams[category['id']]">${category['name']}</h4>
-
-  <ul>
-    <li tal:repeat="exam exams[category['id']]">
-      <a tal:attributes="href request.route_path('exam_enter_points', exam_id=exam.id, tutorial_ids=tutorial_ids)">${exam.name}</a>
-      <a tal:attributes="href request.route_path('exam_enter_points_single', exam_id=exam.id, tutorial_ids=tutorial_ids)">(Interaktiv)</a></li>
-  </ul>
-
+<div class="card">
+  <div class="card-body">
+    <h3 class="card-title">Administrativa</h3>
+
+    <ul>
+      <li><a tal:attributes="href request.route_path('tutorial_results', lecture_id=tutorial.lecture_id, tutorial_ids=tutorial_ids)">Punkteübersicht</a></li>
+      <li><a tal:attributes="href request.route_path('tutorial_email', tutorial_ids=tutorial_ids)">E-Mail an Teilnehmer schreiben</a></li>
+      <li><a tal:attributes="href request.route_path('tutorial_email_preference', tutorial_ids=tutorial_ids)">Status-Emails bestellen/abbestellen</a></li>
+      <li><a tal:attributes='href request.route_path("lecture_email_tutors", lecture_id=tutorial.lecture.id)'>E-Mail an alle Übungsleiter und Assistenten schreiben</a></li>
+      <li tal:condition="request.permissionInfo.has_permission('edit') and len(tutorials) &lt; 2">
+        <a tal:attributes="href request.route_path('tutorial_edit', tutorial_id=tutorial.id)">${names['tutorial']} bearbeiten</a></li>
+      <li tal:condition="request.userInfo.is_tutor_of_tutorials(tutorials)">
+         <a tal:attributes="href request.route_path('tutorial_resign_as_tutor', tutorial_ids=tutorial_ids)">Als ${names['tutor']} zurücktreten</a></li>
```

Comment:
More Icons! Email should have an email icon, Points something with Coins :)

Reply:
@tynsh Do you have a preferred icon font we should use? I'm thinking about adding an icon to every `mailto` link so I don't always confuse them with the weblinks.

Reply:
Well I'm not an expert on fonts at all. As far as I know, font awesome is currently the way to go. And yeah, I think that would be a nice addition

Reply:
Yeah thats one of the things I wanted to work on in the next few days, Font Awesome would probably be the most supported option, but we could also use the Icon set which bootstrap is developing atm... https://icons.getbootstrap.com/

Let me know your opinion about this!